packages/notebook/src/widget.ts

```diff
@@ -1722,8 +1709,8 @@ export class Notebook extends StaticNotebook {
         document.addEventListener('mouseup', this, true);
         document.addEventListener('mousemove', this, true);
       } else if (button === 0 && !shiftKey) {
-        // Prepare to start a drag if we are on the drag region. TODO: If there is no drag, we'll deselect on mouseup.
```

Comment:
Not sure what the author meant with this TODO comment.
@blink1073 does this PR resolves this issue?